### DISCUSSION TITLE
Where can I find KernelFunctionCombinators class?

### DISCUSSION THREAD

**Discussion by lallgaurav** [microsoft/semantic-kernel#7152] [General]
As per below code in one of your blog, we can pipe multiple functions. I'm unable to locate KernelFunctionCombinators, which Nuget package is it part of?

KernelFunction pipeline = KernelFunctionCombinators.Pipe([parseInt32, multiplyByN, truncate, humanize], "pipeline");

**Comment by stephentoub** [comment]
That's just a sample:
https://github.com/microsoft/semantic-kernel/blob/13e3a223af40bafeae4cc95efebcee23592b2efc/dotnet/samples/GettingStarted/Step8_Pipelining.cs#L105-L132

But if you don't need the genericity of such a helper, you can of course just use imperative code to invoke the functions one after the other, using whatever control flow you want.

**Reply by lallgaurav** [reply]
Thanks Stephen for the prompt response. I thought there exist a class named KernelFunctionCombinators.
